sources/packages/forms/src/form-definitions/reportachange.json

```diff
@@ -0,0 +1,4485 @@
+                                "top": "",
+                                "width": "",
+                                "height": ""
+                            },
+                            "type": "panel",
+                            "label": "Panel",
+                            "breadcrumb": "default",
+                            "tabindex": "",
+                            "input": false,
+                            "placeholder": "",
+                            "prefix": "",
+                            "suffix": "",
+                            "multiple": false,
+                            "defaultValue": null,
+                            "protected": false,
+                            "unique": false,
+                            "persistent": false,
+                            "clearOnHide": false,
+                            "refreshOn": "",
+                            "redrawOn": "",
+                            "tableView": false,
+                            "dataGridLabel": false,
+                            "labelPosition": "top",
+                            "description": "",
+                            "errorLabel": "",
+                            "autofocus": false,
+                            "dbIndex": false,
+                            "customDefaultValue": "",
+                            "calculateValue": "",
+                            "calculateServer": false,
+                            "widget": null,
+                            "validateOn": "change",
+                            "validate": {
+                                "required": false,
+                                "custom": "",
+                                "customPrivate": false,
+                                "strictDateValidation": false,
+                                "multiple": false,
+                                "unique": false
+                            },
+                            "allowCalculateOverride": false,
+                            "encrypted": false,
+                            "showCharCount": false,
+                            "showWordCount": false,
+                            "allowMultipleMasks": false,
+                            "tree": false,
+                            "components": [
+                                {
+                                    "label": "Program Details Label",
+                                    "tag": "p",
+                                    "className": "",
+                                    "attrs": [
+                                        {
+                                            "attr": "",
+                                            "value": ""
+                                        }
+                                    ],
+                                    "content": "Program Details",
+                                    "refreshOnChange": false,
+                                    "customClass": "font-weight-bold h4",
```

Comment:
We have different styles for different font styles and sizes, like the below ones. Please try to reuse them.
category-header-small
category-header-medium
category-header-large
label-medium
label-small

Reply:
updated